app/src/main/res/layout/activity_put_pet_adoption.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:orientation="vertical"
+    android:layout_height="match_parent"
+    tools:context=".activity_put_pet_adoption">
+
+    <TextView
+        android:layout_width="142dp"
+        android:layout_height="45dp"
+
+        android:layout_alignTop="@+id/NameEdit"
+        android:layout_alignParentStart="true"
+        android:layout_marginStart="0dp"
+        android:text="Name"
```

Comment:
Resource ya strings

Reply:
as we spoke preivousely, "Name" doesnt need to be resource'd, because we won't be translating this app.